### PR TITLE
[FIX]sale_order_type_automation_stock_voucher

### PR DESCRIPTION
**Pull request by cem-adhoc**
add sale.order methods in sale_order_type_automation_stock_voucher and fix the methods in sale_order_type_automation

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/sale/pull/945).

File: sale_order_type_automation/models/sale_order.py [hunk 1]
```diff
@@ -93,28 +92,18 @@ def run_picking_atomation(self):
...
         res = super().action_confirm()
         # we use this because compatibility with sale exception module
         if isinstance(res, bool) and res:
             # because it's needed to return actions if exists
-            res = self.run_picking_atomation()
+            self.run_picking_atomation()
             self.sudo().run_invoicing_atomation()
             if self.type_id.set_done_on_confirmation:
-                self.action_done()
+                self.locked = True
```
Comment (matiasperalta1) [inline sale_order_type_automation/models/sale_order.py]: Aca es self.action_lock()

**Comment by matiasperalta1** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@cem-adhoc @matiasperalta1 this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#964

**Comment by fw-bot-adhoc** [comment]
@cem-adhoc @matiasperalta1 this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#964

**Comment by fw-bot-adhoc** [comment]
@cem-adhoc @matiasperalta1 this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#964

**Comment by fw-bot-adhoc** [comment]
@cem-adhoc @matiasperalta1 this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#964

**Comment by fw-bot-adhoc** [comment]
@cem-adhoc @matiasperalta1 this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#964

**Comment by fw-bot-adhoc** [comment]
@cem-adhoc @matiasperalta1 this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#964

**Comment by fw-bot-adhoc** [comment]
@cem-adhoc @matiasperalta1 this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#964

**Comment by fw-bot-adhoc** [comment]
@cem-adhoc @matiasperalta1 this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#964

**Comment by fw-bot-adhoc** [comment]
@cem-adhoc @matiasperalta1 this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#964

**Pull request closed without merging**